nrf52-hal-common/src/spim.rs

```diff
@@ -56,6 +57,71 @@ impl_spim_ext!(
 /// - The over-read character is hardcoded to `0`.
 pub struct Spim<T>(T);
 
+impl<T> embedded_hal::blocking::spi::Transfer<u8> for Spim<T> where T: SpimExt
+{
+   type Error = Error;
+
+    fn transfer<'w>(&mut self, words: &'w mut [u8]) -> Result<&'w [u8], Error> {
+        let mut offset:usize = 0;
+        while offset < words.len() {
+            let datalen = min(EASY_DMA_SIZE, words.len() - offset);
```

Comment:
Minor nit-pick: "data" and "len" are two words, so `data_len` would be more idiomatic, I believe. Not a big deal, feel free to leave it.

Reply:
Keeping style of the code is good,I will change.